app/models.py

```diff
@@ -43,6 +43,22 @@ def insert_roles():
     def __repr__(self):
         return '<Role \'%s\'>' % self.name
 
+class IdlingIncident(db.Model):
+    __tablename__ = "incidents"
+    vehicle_id = db.Column(db.Integer, primary_key=True)
+    location = db.Column(db.String(64))
+    date = db.Column(db.DateTime)
+    agency = db.Column(db.String(64))
+    picture = db.Column(db.String(64))
```

Comment:
@nancywong has some experience hosting images on S3. I also think there are some free photo hosting APIs we could check out.